demos/src/demo-kitchen-sink.js

```diff
@@ -1,26 +1,29 @@
-
-function loadScript(name,fn){
+const loadScript = (name, cb) => {
     let head = document.getElementsByTagName('head')[0],
-     script = document.createElement('script');
+        script = document.createElement('script');
 
     script.type = 'text/javascript';
     script.charset = 'utf-8';
     script.defer = true;
     script.src = name + '.js';
     head.appendChild(script);
-    fn && fn();
-}
-window.onload = function(){
+    cb && cb();
+};
 
-    loadScript('/britecharts/scripts/demo-stacked-area');
-    loadScript('/britecharts/scripts/demo-bar');
-    loadScript('/britecharts/scripts/demo-stacked-bar');
-    loadScript('/britecharts/scripts/demo-grouped-bar');
-    loadScript('/britecharts/scripts/demo-donut');
-    loadScript('/britecharts/scripts/demo-line');
-    loadScript('/britecharts/scripts/demo-sparkline');
-    loadScript('/britecharts/scripts/demo-step');
-    loadScript('/britecharts/scripts/demo-brush');
-    loadScript('/britecharts/scripts/demo-scatter-plot');
-    loadScript('/britecharts/scripts/demo-bullet');
+const demoPaths = [
```

Comment:
if we are using es6 this might be better suited as a json file. 
and as a perk you get to add routes without changing code. (only the json file)

Reply:
Makes sense, changing it. Thanks!